### ISSUE TITLE
Duration constructor from Rate

### ISSUE THREAD

**Issue by skasperski** [ros/roscpp_core#30]
There is supposed to be constructor for ros::Duration taking a ros::Rate as param:

```
explicit Duration(const Rate&);
```

I can't see any implementation though and using it gives a linker error.

```
undefined reference to `ros::Duration::Duration(ros::Rate const&)
```

**Comment by dirk-thomas** [comment]
I don't see how this was intended by the original author but the definition of the function is in `ros/impl/duration.h`. If you include that instead / beside `ros/duration.h` your undefined reference at link time should be solved.

**Comment by dirk-thomas** [comment]
Addressed with #32.
